src/Service/RefundService.php

```diff
@@ -206,11 +205,12 @@ public function insertAdyenRefund(
         OrderEntity $order,
         string $pspReference,
         string $source,
-        string $status
+        string $status,
+        int $refundAmount
```

Comment:
Please add this new `refundAmount` parameter to the code above (line 186) which calls this method